pkg/controller/framework/framework_controller_test.go

```diff
@@ -17,20 +17,15 @@ package framework
 
 import (
 	"testing"
-	"time"
-
-	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-var c client.Client
-
-var expectedRequest = reconcile.Request{NamespacedName: types.NamespacedName{Name: "foo", Namespace: "default"}}
-var depKey = types.NamespacedName{Name: "foo-deployment", Namespace: "default"}
-
-const timeout = time.Second * 5
+//var c client.Client
```

Comment:
Can probably just delete instead of commenting out

Reply:
It wasn't clear if this was a reminder to do something... but my preference is to delete it as well... 1 sec